ordnancesurvey/places.py

```diff
@@ -5,18 +5,37 @@
 from dotenv import load_dotenv
 load_dotenv()
 
-
 def get_places_from_search(search):
     """Get places results from search term"""
-
+    fresh_addresses = []
     api_url = "https://api.os.uk/search/places/v1/find?"
     params = {"query": search, 'key': os.environ.get(
         "os_api_key"), 'maxresults': '30'}
     response = requests.get(api_url + urllib.parse.urlencode(params))
-    json = response.json()['results']
 
+    if 'results' in response.json():
+        json = response.json()['results']
+        for value in json:
+            fresh_addresses.append(value['DPA'])
+            
+    return fresh_addresses
+
+def get_building_from_toid(toid):
     fresh_addresses = []
```

Comment:
Ah, I said put this above the linked_ident_api_url because otherwise it's above the doctring. The docstring has to be the first line in a method ("""Get building results from TOID term thru UPRN""")

Reply:
got it! my bad